openedx/core/lib/block_structure/cache.py

```diff
@@ -41,12 +46,13 @@ def add(self, block_structure):
         data_to_cache = (
             block_structure._block_relations,
             block_structure._transformer_data,
-            block_structure._block_data_map
+            block_structure._block_data_map,
         )
         zp_data_to_cache = zpickle(data_to_cache)
         self._cache.set(
             self._encode_root_cache_key(block_structure.root_block_usage_key),
-            zp_data_to_cache
+            zp_data_to_cache,
+            timeout=self.cache_timeout,
```

Comment:
What do you mean?  Are you asking whether this should be configurable via a dynamic django setting?  Or whether it need not be configurable? 
Right now, it's implemented so that any subclasses or clients of `BlockStructureCache` can configure it.